### PR TITLE
Night mode

### PR DESCRIPTION
**Pull request by bahaa96**
### Checklist
- [ ] Unit tests written
- [x] Manually tested
- [ ] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot
capture

### CODE REVIEW

**Comment by naveed-ahmad** [comment]
rebuild

File: src/components/App.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Fragment } from 'react';
 import { asyncComponent } from 'react-async-component';
 import { metrics } from 'react-metrics';
+import cookie from "react-cookie"
```
Comment (mmahalwy) [inline src/components/App.tsx]: nit: eslint should not allow double quotes

File: src/components/App.tsx [hunk 2]
```diff
@@ -12,15 +13,22 @@ const SmartBanner = asyncComponent({
...
-    <Navbars />
-    <SmartBanner title="The Noble Quran - القرآن الكريم" button="Install" />
-    <NoScript />
-    <Routes />
-    <Footer />
-  </Fragment>
-);
+const App = () => {
+  const { isNightMode } = cookie.load("options");
+  if(__CLIENT__)
```
Comment (mmahalwy) [inline src/components/App.tsx]: nit: there should be a space between `if` and the paranthesis

File: src/components/SmartBanner/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components"
```
Comment (mmahalwy) [inline src/components/SmartBanner/Button.tsx]: Hmm... this is nightmode no? smart banner?

File: src/components/Verse.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { connect } from "react-redux";
```
Comment (mmahalwy) [inline src/components/Verse.tsx]: This is already wrapped by `VerseContainer`. We don't need to do this

File: src/components/audioplayer/Wrapper.tsx [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+const Wrapper = styled.div<{ isNightMode: boolean }>`
   position: fixed;
   bottom: 0;
   display: block;
   user-select: none;
   height: auto;
   z-index: 1;
   padding: 20px 20px 10px;
-  background: ${({ theme }) => theme.colors.white};
+  background: ${({ theme, isNightMode }) => isNightMode ? theme.colors.tuatara : theme.colors.white};
```
Comment (mmahalwy) [inline src/components/audioplayer/Wrapper.tsx]: very nice!

File: src/components/settings/FontSizeOptions.tsx [hunk 6]
```diff
@@ -14,6 +14,10 @@ const List = styled.ul`
   display: table;
   width: 100%;
   padding: 15px 0;
+  &:hover {
+    background-color: #F5F5F5;
```
Comment (mmahalwy) [inline src/components/settings/FontSizeOptions.tsx]: what are these colors? Can they be added to theme?

Comment (bahaa96) [reply src/components/settings/FontSizeOptions.tsx]: I'll Clean up this PR

**Comment by mmahalwy** [comment]
Hmm.. I am concerned that eslint did not complain about the double quotes. That should be fixed

**Pull request closed without merging**